backend/src/services/clientServerService.js

```
@@ -1,184 +1,64 @@
-         data: {
-            client_id,
-            client_secret, // Only returned during registration
-            app_name,
-            assigned_schema_name,
-            allowed_return_urls,
-            client_mode,
-         },
+         data: createdClientServer,
       };
    } catch (error) {
       throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)